Tests/test_Entrez_parser.py

```diff
@@ -2291,7 +2291,7 @@ def test_integer_none(self):
         self.assertEqual(record["BondChiralUndefCount"], 0)
         self.assertEqual(record["IsotopeAtomCount"], 0)
         self.assertEqual(record["CovalentUnitCount"], 1)
-        self.assertEqual(record["TautomerCount"], None)
+        self.assertIsNone(record["TautomerCount"])
```

Comment:
I didn't spot until the automated testing failed:

```
======================================================================
FAIL: test_integer_none (test_Entrez_parser.ESummaryTest)
Test parsing ESummary XML where an Integer is not defined.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\projects\biopython\Tests\test_Entrez_parser.py", line 2294, in test_integer_none
    self.assertIsNone(record["TautomerCount"])
AssertionError: NoneElement(attributes={}) is not None
----------------------------------------------------------------------
```

This one is interesting - https://github.com/biopython/biopython/blob/master/Bio/Entrez/Parser.py#L55 - we get an object which acts like ``None`` but isn't the special ``None`` value.

Put this back to using ``self.assertEqual`` please, but perhaps add the ``# noqa: A502`` comment to stop flake8 complaining.